### PR TITLE
Fix issues in the booking package

### PR DESCRIPTION
**Pull request by wolfrednicolas**

### CODE REVIEW

File: database/factories/BookingFactory.php [hunk 1]
```diff
@@ -32,9 +32,21 @@ public function definition()
...
-            'booking_category_id'   => BookingCategory::factory()->create()->id,
-            'rate_id'               => Rate::factory()->create()->id,
-            //Todo: Add morph factories
+            'booking_category_id'   => randomOrCreate(app('booking_category')),
+            'rate_id'               => randomOrCreate(app('rate')),
+            'experience_id'         => $this->faker->numberBetween(0, 10),
+            'experience_type'       => 'Game',
+            'order_id'              => $this->faker->numberBetween(0, 10),
+            'order_type'            => 'Order',
+            'booking_status_id'     => '1',
```
Comment (wolfrednicolas) [inline database/factories/BookingFactory.php]: @sl0wik I have a question about the `'booking_status_id'`

I saw the migration definition and saw this

image

But I the whimsical there is this

image


I dont know very well how is the intention having the `Status::publishStatuses`

Comment (chx2) [reply database/factories/BookingFactory.php]: @wolfrednicolas check out https://github.com/tipoff/bookings/pull/47, the decision was made to use Statuses package instead

Comment (wolfrednicolas) [reply database/factories/BookingFactory.php]: thanks @chx2

**Comment by sl0wik** [comment]
@drewroberts we can merge it to make tests green. @wolfrednicolas you can fix the rate in next PR.

**Review by drewroberts — APPROVED** [review]
Muchas gracias, amigos! 🌴

**Pull request merged**